bibtex-actions.el

```diff
@@ -38,10 +38,59 @@
 
 (require 'bibtex-completion)
 
+;;; Variables
+
 ;; REVIEW: is this the correct way to ensure we use the custom separator in
 ;;         'bibtex-actions--completing-read'?
 (defvar crm-separator)
 
+(defcustom bibtex-actions-rich-ui t
+  "Adds prefix symbols or icons and secondary metadata for suffix UI.
+Affixation was first introduced in Emacs 28, and will be ignored
+in previous versions."
+  :group 'bibtex-actions
+  :type 'boolean)
+
+(defcustom bibtex-actions-suffix-display-formats
```

Comment:
Finally, this is more an FYI: I added this config option for the suffix; copied from the main display defcustom for bibtex-completion.